src/apps/concepts/components/EnhancedTableToolbar.tsx

```diff
@@ -70,7 +70,7 @@ export const EnhancedTableToolbar = (props: EnhancedTableToolbarProps) => {
       )}
       {numSelected > 0 ? (
         <>
-          {showAddConcepts ? null : (
+          {!showAddConcepts  ? null : (
```

Comment:
This line should read:

```ts
{showAddConcepts ? null : (
```

We need to change the conditions under which `showAddConcepts` is true, not just negate it (if we keep this in, this will show the `add concepts` part where we don't want to do that, i.e., where the user didn't start from their own dictionary).